### DISCUSSION TITLE
How does DPDATA understand MD-Singlepoint-Relax in CP2K

### DISCUSSION THREAD

**Discussion by EYErbil** [deepmodeling/dpdata#692] [Q&A]
Hello, my cp2k output has only the .out file, and I convert this data to an input string, with different phases and different simulations, some of them are single point, some of them are relaxation, some of them are md.
I want to train with all, same model,  I run a script with : 
```python
os.system(f"dpdata {output_file} -i cp2k/output -o deepmd/npy -O {data_dir}")
``` 
Is there any way I can understand if I am doing the right thing, and deepmd can differentiate between different calculations?

**Comment by njzjz** [comment]
Training data should have consistent DFT configurations, such as converge accuracy and energy cutoff. The sources for atomic coordinates are not important, but to my knowledge, sometimes people use different accuracies for MD and single-point calculation. In this case, these coordinates should be relabeled.

**Reply by EYErbil** [reply]
Some has different number of atoms too, this does not matter too I suppose
